File: example15/HelloJNI/app/src/main/java/com/jikexueyuan/hellojni/Tools.java
package com.jikexueyuan.hellojni;

import android.content.Context;
import android.widget.Toast;

/**
 * Created by dej on 2016/11/26.
 */

public class Tools {

    public static Context context = null;

    public static void init(Context context) {
        Tools.context = context;
    }

    public static void showToast(String msg) {
        Toast.makeText(context, msg, Toast.LENGTH_SHORT).show();
    }
}

File: example3/HelloKotlin/app/src/main/java/com/jikexueyuan/hellokotlin/Tools.kt
package com.jikexueyuan.hellokotlin

import android.content.Context
import android.widget.Toast

/**
 * Created by dej on 2016/9/13.
 */
class Tools {

    companion object {
        fun showToast(c:Context, msg:String) {
            Toast.makeText(c, msg, Toast.LENGTH_SHORT).show()
        }
    }
}